tests/storage/checkups/conftest.py

```diff
@@ -194,13 +196,19 @@ def ocs_rbd_non_virt_vm_for_checkups_test(admin_client, checkups_namespace):
 
 
 @pytest.fixture()
-def broken_data_import_cron(golden_images_namespace):
+def cleaned_up_broken_data_source(golden_images_namespace):
```

Comment:
Why do you need a separate fixture? If DataImportCron creates it, you should  clean it up in broken_data_import_cron.

Reply:
Right, moving it to the broken_data_import_cron